Create.c

```diff
@@ -1371,6 +1370,8 @@ int Create(struct supertype *st, struct mddev_ident *ident, int subdevs,
 	if (mdfd >= 0)
 		close(mdfd);
 
+	close_fd(&container_fd);
```

Comment:
by chance, you can change this mdfd check upper :)

Reply:
@mtkaczyk what do you mean here?

Reply:
```c
	close_fd(&mdfd);
	close_fd(&container_fd);
```

Reply:
gotcha.